src/components/SearchAndFilter.js

```diff
@@ -156,11 +167,4 @@ class SearchAndFilter extends React.Component {
   }
 }
 
-const styles = {
-  formBody: {
-    justifyContent: "center",
-    display:"flex"
-  }
-}
-
 export default SearchAndFilter;
```

Comment:
super nit - it's usually good to have a newline at the end of the file. It helps with diffing mechanisms and consistency in copying/moving files around in OS's

Reply:
done